test/quick_test_helper.rb

```diff
@@ -1,7 +1,4 @@
-if ENV['RAILS_ENV']
-  def rails_require(*)
-  end
-else
+if ENV['RAILS_ENV'] == "test"
```

Comment:
Huh.

It looks like [`lib_require` was added after `rails_require`](https://github.com/stitchfix/resque-brain/commit/a2232d25b740ea12b3a515c0c7c02bceed050c05#diff-b685edaaf0b760b12b2dd082e873bdea), and [`lib_require` is only used in tests](https://github.com/stitchfix/resque-brain/search?utf8=%E2%9C%93&q=lib_require).

If you add `puts ENV['RAILS_ENV']`, does it print anything while running `bundle exec rake test`?  Thanks!

```ruby
puts ENV['RAILS_ENV']
if ENV['RAILS_ENV']
  def rails_require(*)
  end
else
  $: << File.expand_path(File.join(File.dirname(__FILE__),'..'))
  def rails_require(file)
    require "app/#{file}"
  end

  def lib_require(file)
    require "lib/#{file}"
  end
end
```

```
❯ bundle exec rake test
test
rake aborted!
NoMethodError: undefined method `lib_require' for main:Object
/Users/simeonwillbanks/Code/resque-brain/test/lib/monitoring/failed_job_check_test.rb:6:in `<top (required)>'
Tasks: TOP => test:run
(See full trace by running task with --trace)
```